nixos/modules/services/networking/twingate.nix

```diff
@@ -9,75 +9,21 @@ in {
 
   options.services.twingate = {
     enable = mkEnableOption "Twingate Client daemon";
-
-    package = mkOption {
-      type = types.package;
-      default = pkgs.twingate;
-      defaultText = "pkgs.twingate";
-      description = "The package to use for Twingate";
-    };
   };
 
   config = mkIf cfg.enable {
 
     networking.firewall.checkReversePath = lib.mkDefault false;
 
-    environment.systemPackages = [ cfg.package ]; # for the CLI
-    systemd.packages = [ cfg.package ];
+    environment.systemPackages = [ pkgs.twingate ]; # for the CLI
+    systemd.packages = [ pkgs.twingate ];
 
     system.activationScripts.twingate = stringAfter ["etc"] ''
       mkdir -p '/etc/twingate'
 
-      cp -r -n ${cfg.package}/etc/twingate/. /etc/twingate/
-    '';
-
-    # copy from twingate.service
-    # passed as raw text to reset ExecStart, because of weird error:
-    # twingate.service: Service has more than one ExecStart= setting...
-    systemd.units."twingate.service".text = ''
```

Comment:
I didn't get where twingate.service comes from? From origin package that is used during conversion?

Reply:
Yes, looks like it will use existing and no need of duplicating it here (I checked and it really worked like this)